tests/jq.test

```diff
@@ -1765,3 +1765,15 @@ false
 1
 1
 
+# abs, fabs, length
+abs
+"abc"
+"abc"
```

Comment:
How should be expected to work against null and booleans? Emits error in the current implementation but should be as it is?

Reply:
And how should it work with strings, arrays, and objects as inputs?  Currently it passes them through.  That's very strange, but it's an artifact of strings, arrays, and objects sorting after numbers in jq, so this `abs/0` doesn't want to negate them, so there's no error, but `null`, `false`, and `true` sort ahead of numbers, so `abs/0` does want to negate them, and that then raises an error.

We have some options

- leave as-is
- the `abs` of all non-number values should be the identity
- the `abs` of all non-number values should be an error